dbms/src/Functions/array/arrayFill.cpp

```diff
@@ -0,0 +1,126 @@
+#include <DataTypes/DataTypesNumber.h>
+#include <Columns/ColumnsNumber.h>
+#include "FunctionArrayMapped.h"
+#include <Functions/FunctionFactory.h>
+
+
+namespace DB
+{
+
+template <bool Reverse>
```

Comment:
style, `Reverse -> reverse` is better ?

Reply:
We can find all of the 4 major styles in the repo...
Anyway, `reverse` is good according to the [guide](https://clickhouse.yandex/docs/en/development/style/)